docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -535,6 +535,28 @@ else [`clusterName`](#clusterName) is used. Default port is 3024.
   tunnelPublicAddr: ["loadbalancer.example.com:3024"]
   ```
 
+## `trustXForwardedFor`
+
+| Type   | Default value | Required? | `teleport.yaml` equivalent            |
+|--------|---------------|-----------|---------------------------------------|
+| `bool` | `false`       | no        | `proxy_service.trust_x_forwarded_for` |
+
+`trustXForwardedFor` enables the Proxy service to take client source IPs from
+the "X-Forwarded-For" headers for web APIs.
+
+<Admonition type="warning">
+In order to prevent IP spoofing, it is important to ensure that the Proxy
+service is placed behind a layer 7 load balancer or reverse proxy that sets
+client IPs in the "X-Forwarded-For" headers. Requests with more than one IPs
+will also be rejected.
```

Comment:
Same comment as above re "also".

Reply:
helm chart doc is reverted as the related helm change https://github.com/gravitational/teleport/pull/28420 is cancelled. The new service config will be "embedded" in the new ingress change https://github.com/gravitational/teleport/commit/446bd1c05e318ecb68beb49b9f6cea2b9a4ccc47